tests/RazorLight.Tests/Generation/RazorSourceGeneratorTest.cs

```diff
@@ -120,13 +120,14 @@ public async Task GetImports_Returns_EmptyCollection_On_Empty_Project_WhenResolv
 
 
         [Fact]
-        public void GenerateCode_ByKey_Throws_OnEmpty_Project()
+        public async void GenerateCode_ByKey_Throws_OnEmpty_Project()
```

Comment:
The "async void" tests are generally discouraged. 

```suggestion
        public async Task GenerateCode_ByKey_Throws_OnEmpty_Project()
```